### PR TITLE
update Dockerfile for new deco and SSM support

### PR DESCRIPTION
**Pull request by damnski**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -26,18 +26,12 @@ WORKDIR /app
 COPY --from=build-env /app/api.out /app/api
 RUN chmod 555 /app/api
 
-RUN apk add --no-cache bash ca-certificates wget python gettext && \
-    wget -nv "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" && \
-    unzip awscli-bundle.zip && \
-    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws && \
-    /usr/local/bin/aws --version
```
Comment (fishnix) [inline docker/Dockerfile:33]: You're going to need some of those packages I think

**Review by fishnix — COMMENTED** [review]
Just the one comment.

File: docker/Dockerfile (hunk 1 above)
Comment (FishNecks) [reply docker/Dockerfile:33]: Yeah, add the line `RUN apk add --no-cache bash ca-certificates` after the line you deleted above.

**Comment by FishNecks** [comment]
I think you'll also need to change the `import_config.sh` file to use SSM instead of S3.

File: docker/Dockerfile (hunk 1 above)
Comment (damnski) [reply docker/Dockerfile:33]: :+1:

**Comment by fishnix** [comment]
I think this is good - if you're unsure, cut a prerelease and test 👍

**Pull request merged**